GIT_AND_GITHUB.md

```diff
@@ -42,26 +42,39 @@ refactor/make-it-awesome
 
 Use `--no-ff` when you merge to `master`.
 
-### Merging
-
 It's okay to just push to `master` if it's a quick fix and you really need
 it out the door. Otherwise, make it a pull request.
 
+### Pull requests
+
 Pull request descriptions should be concise and well written. The merger should
 be able to copy this description straight into the release notes instead of
 figuring out what changed or was fixed.
 
-#### Who merges the pull request?
+Besides a clear description more information could be helpful, for example:
+
+- If it's static UI, a screenshot would do. 
+- If it's an interaction a gif would help a lot. A good tool to make gifs is [Licecap](http://www.cockos.com/licecap/).
+- If it's a bug, steps to reproduce are useful to understand the context.
+
+#### When to merge a pull request?
+
+Before merging a pull request your code has to be reviewed so we can learn from you, 
+point out your silly mistakes and/or post the sufficient amount of gifs. The reviewing
+process is important, is better for you to have another person backing you up, more eyes
+means less bugs and more consistency, trust us.
+
+When the reviewer signs off confirming that the changes are ready, feel free to 
+merge your pull request. Avoid merging before receiving a confirmation, even for 
+"final fixes" and "last touches". If several developers are involved in the project 
+one confirmation should be enough, you can always submit another PR or file an issue.
 
-Core contributors have the final say on what gets merged into their codebase,
-so they get to click the button. That could be you, but if you're working with
-someone else it should probably be them. The important thing is that someone else
-gets to look it over so we can learn from you, point out your silly mistakes and/or
-post the sufficient amount of gifs.
+As a reviewer when signing off on an issue, be sure to be as explicit and unambiguous as possible.
 
-Once the pull request is accepted, the person who merged should delete the branch.
+After you have merged, you should delete the branch and smile, the smile is critical 
+part of the process, so don't forget this.
```

Comment:
and smile**.** The smile is critical  part of the process, so don't forget this.